x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.test.tsx

```diff
@@ -15,7 +15,7 @@ import { KSPM_POLICY_TEMPLATE } from '../../../../common/constants';
 import {
   DASHBOARD_TABLE_COLUMN_SCORE_TEST_ID,
   DASHBOARD_TABLE_HEADER_SCORE_TEST_ID,
-} from '../../findings/test_subjects';
+} from '../../configurations/test_subjects';
```

Comment:
I dont think dashboard test subjects should be imported from findings/configurations test subject file. not sure why it was there at first place, probably a mistake

Reply:
fixed, good catch